src/components/UpcomingEventCard/EventCard.style.js

```diff
@@ -13,7 +13,10 @@ const UpcomingEventsWrapper = styled.div`
 		height: 39.5rem;
 		transition: all 0.3s;
 	}
-
+	.blog-slider_img{
+		max-height: 80rem; 
```

Comment:
You also need to provide the max-height for the `img` tag under `gatsby--image-wrapper` under this property itself.

Reply:
That is:
```
.blog-slider_img {
    ......
    ......
    .gatsby-image-wrapper {
         img {
              max-height: <some-value>
          }
    }
}
```
Do check if the intermediary `.gatsby-image-wrapper` is needed or not.
Also, reduce the value of `max-height` to around `22rem`

Reply:
I think there is no need for .gatsby-image-wrapper and seperate img tag I have check and nothing change is there

Reply:
Do you mean we don't need to update the above CSS at all?
If so, try to reduce the max-height of your current CSS to `20rem` and see if the height updates (it shouldn't)

Reply:
Yes you are right it is not updating let me correct it

Reply:
Changes I made
.blog-slider_img{
		max-height: 30rem; 
		object-fit: cover;
		.gatsby-image-wrapper {
			img {
				 max-height: 23rem;
			 }
	   }
	}
And here is the result
![Screenshot from 2022-03-10 12-38-12](https://user-images.githubusercontent.com/59443454/157607953-4c96afa7-5a50-489a-9836-ace5931bc6d0.png)

Reply:
Keep values for both the classes same i.e, 25rem (considering the largest image) .